src/heif.imageio/CMakeLists.txt

```diff
@@ -3,6 +3,33 @@
 # https://github.com/OpenImageIO/oiio
 
 if (Libheif_FOUND)
+   if (LINKSTATIC)
+    if (WIN32)
+        set (_static_suffixes .lib .a)
+    else ()
+        set (_static_suffixes .a)
+    endif ()
```

Comment:
Can this be simplified? If you just do .lib and .a, and not need the `if`, so what? It won't find .lib in the libraries on non-Windows, so it doesn't hurt to search for it.